### PR TITLE
Update design for homepage in preperation for 2.0 release

### PR DESCRIPTION
**Pull request by whmii**
mew

### CODE REVIEW

File: source/assets/stylesheets/pages/_index.scss [hunk 1]
```diff
@@ -16,36 +34,51 @@
...
   }
+
+  img {
+    display: block;
+  }
+}
+
+.header-image {
+  max-width: $row-width / 2;
+  margin: 0 auto;
```
Comment (ColeTownsend) [inline source/assets/stylesheets/pages/_index.scss]: nit picking, could you do null?

File: source/assets/stylesheets/pages/_index.scss [hunk 2]
```diff
@@ -140,19 +173,25 @@
...
 
 .page-navigation {
-  border-radius: 0 0 $base-border-radius $base-border-radius;
   float: right;
   padding: 0.125em 0.5em;
-  position: relative;
+  position: absolute;
+  top: 0.125rem;
+  right: 0.125rem;
+  @media #{$small} {
```
Comment (ColeTownsend) [inline source/assets/stylesheets/pages/_index.scss]: I usually like a space in here

File: source/assets/stylesheets/patterns/_signup.scss [hunk 3]
```diff
@@ -10,6 +11,11 @@
     padding: 3em;
   }
 
+
```
Comment (ColeTownsend) [inline source/assets/stylesheets/patterns/_signup.scss]: extra line

File: source/assets/images/core-016-a.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...,0.358l0.437,-0.711c-0.466,-0.234 -0.94,-0.453 -1.421,-0.659ZM119.9,37.087c-0.765,0.064 -1.521,0.162 -2.265,0.293l1.132,2.746c0.173,-0.141 0.393,-0.226 0.633,-0.226c0.182,0 0.353,0.049 0.5,0.134l0,-2.947ZM51.574,36.93l2.772,-4.43l-0.824,-0.55l-2.776,4.515c0.279,0.151 0.555,0.306 0.828,0.465ZM115.583,33.714l-1.164,-2.823l-1.11,0.506c0.034,0.064 0.064,0.132 0.091,0.203l0.811,2.461c0.453,-0.127 0.91,-0.242 1.372,-0.347ZM108.409,18.235l0.509,1.537l0.626,-0.209l0.885,2.657l-0.629,0.21l2.726,8.225l1.641,-0.748l-4.267,-10.307c-0.415,-0.947 -0.794,-1.299 -1.491,-1.365ZM106.3,18.358l-46.985,6.687l0.185,0.555c0,0.2 0.3,0.3 0.5,0.3l46,-6.6c0.3,-0.1 0.5,-0.4 0.4,-0.7l-0.062,-0.2l-0.038,0l0,-0.042ZM109,10l-1,0l0.3,0.3c0.3,0.3 0.3,0.7 0.2,1c-0.1,0.3 -0.4,0.5 -0.8,0.5l-1.6,0l0.428,1.369l0.616,-0.206l0.885,2.657l-0.666,0.222l0.171,0.547l1.166,-0.389l0.3,0.9l-0.918,0.348l0.259,0.782c0.61,0.031 1.042,0.223 1.422,0.722c-0.591,-1.78 -0.775,-3.021 -0.263,-3.852c0.5,-0.7 1.6,-1.1 3.5,-1.1c1.3,0 1.9,-0.7 1.9,-1.4c0,-0.7 -0.6,-1.4 -1.9,-1.4c-1.169,0 -2,-0.169 -2.522,-0.478l-0.478,0.478l-1,-1ZM57,12.5l1.333,-0.5l-2.133,0c-0.1,0 -0.3,0 -0.4,0.1l-0.357,0.121l0.557,0.279l1,0ZM107,9" style="fill:#fff;"/></svg>
```
Comment (tysongach) [inline source/assets/images/core-016-a.svg]: Is this simply a visual element? Can we add the proper `role` and ARIA attributes, and give it a proper `title`?

File: source/assets/images/github.svg [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<svg width="100%" height="100%" viewBox="0 0 60 60" version="1.1" xmlns="http://www.w3.org/2000/svg">
```
Comment (tysongach) [inline source/assets/images/github.svg]: Same note here, regarding a11y.

File: source/index.haml [hunk 6]
```diff
@@ -5,25 +5,16 @@ title: Foundry ∞
...
-        %p
-          SVGs are amazing. Finding new and interactive ways to share visual information, art, and fun is awesome and SVGs let us do this in ways that aren’t possible in raster art.
-        %p
-          Foundry is a new way of utilizing scalable vector graphics, giving designers the opportunity to contribute and collaborate on families of spot vector illustrations and icons and make them available for download/purchase. No more hunting around google for ten minutes to find just the right icon or wasting time recreating an iPhone when you know it’s already been done a thousand times.
-          %button.subtitle-linktodescription.less{ data: { toggle: "less" } }
-            less
-    %nav.header-linkto
-      = link_to "Gallery", "#gallery", class: "header-linktogallery button"
+      %h2
+        = succeed '.' do
```
Comment (tysongach) [inline source/index.haml]: Looks like we are using double quotes elsewhere in this file; do the same here?

**Pull request merged**